tests/class-all-test.php

```diff
@@ -813,4 +815,43 @@ public function test_get_current_url( $force_https, $url, $expected ) {
 		$res = self::$parsely->get_current_url();
 		self::assertStringStartsWith( $expected, $res );
 	}
+
```

Comment:
See where #272 is heading, and move these tests to their own file. You can then avoid / set sensible defaults in the `setUp()` or per test method.